packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.Props.ts

```diff
@@ -54,9 +54,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
 
   /**
    * Whether the ComboBox auto completes. As the user is inputing text, it will be suggested potential matches from the list of options. If
-   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style. Defaults to false.
+   * the combo box is expanded, this will also scroll to the suggested option, and give it a selected style.
+   *
+   * @default "off"
    */
-  autoComplete?: string;
+  autoComplete?: 'on' | 'off';

```

Comment:
Definitely would be more clear to specify valid options, but going back to the [previous discussion](https://github.com/OfficeDev/office-ui-fabric-react/pull/3042) of having this prop match the HTMLAttributes type definition, the definition is only `string`. Would there be any conflicts here if we specify on or off as the only valid strings? Maybe the type definition itself should be updated? @jspurlin @kristoferbrown